### PR TITLE
EDGEPOD-334 : NSA MME:S6 Diameter Protocol Layer Support

### PR DESCRIPTION
**Pull request by clstefi**
This activity includes the support of following AVPs in s6a-app:

1. Supported Features with its child AVPs namely Feature-List-ID and Feature List in ULR and ULA 
   messages.
2. Extended-Max-Requested-BW-UL and Extended-Max-Requested-BW-DL , part of           
    Subscription-Data->AMBR in ULA.

### CODE REVIEW

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

File: src/s6a/handlers/s6a_req_handler.c [hunk 1]
```diff
@@ -115,7 +116,24 @@ send_FD_ULR(struct s6a_Q_msg *aia_msg, char imsi[])
 	val.os.data = (unsigned char*)aia_msg->tai.plmn_id.idx;
 	val.os.len = 3;
 	add_fd_msg(&val, g_fd_dict_objs.visited_PLMN_id, &fd_msg);
-
+	
+	/*AVP: Supported Features*/
```
[the remark this answers is not in the document]
Comment (clstefi) [reply src/s6a/handlers/s6a_req_handler.c]: Addressed in the commit https://github.com/omec-project/Nucleus/pull/19/commits/1f4a62c05afb4a718b4418c0d6128b00d247185d

**Comment by asreek** [comment]
@thakurajayL 
Hi Ajay,
Hari was not able to merge this because of the pending CI job. Can you please merge this?

Thanks,
Anjana

**Comment by thakurajayL** [comment]
retest this please

**Pull request merged**